BRANDING.md

```diff
@@ -0,0 +1,11 @@
+# Branding
+
+Kubernetes-based Event Driven Autoscaling is a CNCF Sandbox project and follows to the [CNCF/LF trademark usage](https://www.linuxfoundation.org/trademark-usage/).
+
+When referring to Kubernetes-based Event Driven Autoscaling in short, it should be called "KEDA" with full upper-casing.
+
+## Logos
+
+We provide various logos to use and can be found on the [official CNCF Artwork](https://github.com/cncf/artwork/blob/master/examples/sandbox.md#keda-logos) page.
+
+For online usage, we recommend using the shadow while for print it is recommended to use our simple approach.
```

Comment:
```suggestion
For online usage, we recommend using the shadow while for print it is recommended to use our simple approach.
```
Should we add links to those versions? This should increase accessibility

Reply:
I've used a different approach PTAL